scanner/django.go

```diff
@@ -41,6 +44,69 @@ func configureDjango(sourceDir string, config *ScannerConfig) (*SourceInfo, erro
 
 	vars := make(map[string]interface{})
 
+	var pythonSupported string = "3.7" // https://devguide.python.org/versions/#supported-versions
+	var pythonVersion string = "3.10" // Keep the default version "3.10" (hardcoded on the Dockerfile)
+	var pythonVersionOutput string = "Python 3.10.0"
+	var pythonVersionFound bool = false
```

Comment:
Go has a bit of type inference. So you can do `pythonSupported := "3.7"` instead.